cmd/sebak/cmd/wallet/payment.go

```diff
@@ -214,8 +223,12 @@ func makeTransactionCreateAccount(kpSource keypair.KP, kpDest keypair.KP, amount
 /// Returns:
 ///  `sebak.Transaction` = The generated `Transaction` to do a payment
 ///
-func makeTransactionPayment(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, seqid uint64) transaction.Transaction {
+func makeTransactionPayment(kpSource keypair.KP, kpDest keypair.KP, amount common.Amount, sender block.BlockAccount) transaction.Transaction {
 	opb := operation.NewPayment(kpDest.Address(), amount)
+	fee := common.BaseFee
+	if sender.Linked != "" {
+		fee = common.Amount(0)
```

Comment:
How about `fee = common.FrozenFee` and `FrozenFee = Amount(0)` ?

Reply:
LGTM!